install/uninstall.go

```diff
@@ -141,3 +145,29 @@ func (k *K8sUninstaller) waitForPodsToBeDeleted(ctx context.Context) error {
 		}
 	}
 }
+
+func (k *K8sUninstaller) enableCNIUninstall(ctx context.Context) {
+	pods, err := k.client.ListPods(ctx, k.params.Namespace, metav1.ListOptions{LabelSelector: defaults.AgentPodSelector})
+	if err != nil {
+		k.Log("❌ Failed to enable cni cleanup: %v", err)
+		return
+	}
+	wp := workerpool.New(k.params.WorkerCount)
```

Comment:
We need to shutdown the workerpool eventually to avoid leaking its "run" goroutine. Also we can provide the base context since we have been given one
```suggestion
	wp := workerpool.NewWithContext(ctx, k.params.WorkerCount)
	defer wp.Close()
```

Reply:
aha, good point!